### DISCUSSION TITLE
How can I access the ArchWSL /home/myUser from a dual boot?

### DISCUSSION THREAD

**Discussion by lucasrdrgs** [yuk7/ArchWSL#235] [Q&A] [answered]
I can mount my Windows partition when I'm in Linux (I use Arch). However, I have some files in my Windows ArchWSL installation that I would like to transfer to the Linux partition. How can I do so from the Linux terminal? Where are those files located, assuming `\\wsl$` wont work since I'm not on Windows?

**Comment by yuk7** [comment, marked as answer]
Hi @lucasrdrgs 
ArchWSL's root file system is located in ext4.vhdx.
If you are using AppX, it's in `C:\Users\{Your-User-Name}\AppData\Local\Packages\yuk7.archwsl_35zwpb4sx6e50\LocalState`.

You can use ndb and qemu-ndb on native Linux to recognize it as a block device.
```
sudo qemu-nbd -c /dev/nbd0 ext4.vhdx
```
and you can mount it as ext4 partition
```
mount -o ro,nouser /dev/nbd0 /mnt/mountpoint
```
